prover/zkevm/prover/statemanager/accumulator/define.go

```diff
@@ -373,6 +376,17 @@ func (am *Module) checkDelete() {
 	am.comp.InsertGlobal(am.Round, am.qname("IS_DELETE_BOOLEAN"), expr3)
 }
 
+func (am *Module) checkUpdate() {
```

Comment:
Again, `ifaces.ColumnAsVariable` should not be needed here as `symbolic.Sub` accepts columns directly

Reply:
But the constraint itself is ok

Reply:
Thanks, I removed `ifaces.ColumnAsVariable` from everywhere. To avoid merge conflict, I suggest to merge this PR at the last.